javascript/javascript3/week3/shoppingcart.js

```diff
@@ -0,0 +1,101 @@
+class Product {
+    constructor(name, price) {
+      this.name = name;
+      this.price = price;
+    }
+    convertToCurrency(currency) {
+      if (currency === "USD") {
+        return this.price * 0.157;
+      } else if (currency === "INR") {
+        return this.price * 11.60;
+      } else if (currency === "EUR") {
+        return this.price * 0.13;
+      } else {
+        return "enter currency in (USD,INR,EUR)";
+      }
+    }
+  }
+  
+  class ShoppingCart {
+    constructor() {
+      this.products = [];
+    }
+  
+    addProduct(product) {
+      this.products.push(product);
+    }
+  
+    removeProduct(product) {
+      if (this.products.length !== 0) {
+        this.products.splice(this.products.indexOf(product), 1);
+      }
+    }
+  
+    searchProduct(productName) {
+      return this.products.filter((data) => data.name.toLowerCase() === productName.toLowerCase());
+    }
+  
+    getTotal() {
+      return this.products.map((data) => data.price).reduce((a, b) => a + b, 0);
+    }
+  
+    getUser() {
```

Comment:
this function is called getUser, but it actually gets AND renders the user. Better to keep these two things separate.